nav2_util/include/nav2_util/simple_action_server.hpp

```diff
@@ -62,6 +65,8 @@ class SimpleActionServer
       server_active_ = true;
     }
 
+    server_timeout_ = std::chrono::milliseconds(2000);
```

Comment:
This should be a parameter in the constructor that defaults to 2s. I think that's really high, it should be closer to 0.5 seconds or maybe even 1s.

Reply:
Oh, I was experimenting with pulling it from a node parameter, decided not to, and forgot to move it to the constructor...